iamidentityv1/iam_identity_v1_examples_test.go

```diff
@@ -426,6 +438,292 @@ var _ = Describe(`IamIdentityV1 Examples Tests`, func() {
 			Expect(err).To(BeNil())
 			Expect(response.StatusCode).To(Equal(204))
 		})
+		It(`CreateProfile request example`, func() {
+			fmt.Println("\nCreateProfile() result:")
+			// begin-create_profile
+
+			createProfileOptions := iamIdentityService.NewCreateProfileOptions(profileName, accountID)
+			createProfileOptions.SetDescription("Example Profile")
+
+			profile, response, err := iamIdentityService.CreateProfile(createProfileOptions)
+			if err != nil {
+				panic(err)
+			}
+			b, _ := json.MarshalIndent(profile, "", "  ")
+			fmt.Println(string(b))
+			profileId = *profile.ID
+
+			// end-create_profile
+
+			Expect(err).To(BeNil())
+			Expect(response.StatusCode).To(Equal(201))
+			Expect(profile).ToNot(BeNil())
+			Expect(profileId).ToNot(BeNil())
+		})
+		It(`GetProfile request example`, func() {
+			fmt.Println("\nGetProfile() result:")
+			// begin-get_profile
+
+			getProfileOptions := iamIdentityService.NewGetProfileOptions(profileId)
+
+			profile, response, err := iamIdentityService.GetProfile(getProfileOptions)
+			if err != nil {
+				panic(err)
+			}
+			profileEtag = response.GetHeaders().Get("Etag")
+			b, _ := json.MarshalIndent(profile, "", "  ")
+			fmt.Println(string(b))
+
+			// end-get_profile
+
+			Expect(err).To(BeNil())
+			Expect(response.StatusCode).To(Equal(200))
+			Expect(profile).ToNot(BeNil())
+			Expect(profileEtag).ToNot(BeEmpty())
+		})
+		It(`ListProfiles request example`, func() {
+			fmt.Println("\nListProfiles() result:")
+			// begin-list_profile
+
+			listProfileOptions := iamIdentityService.ListProfileOptions()
+			listProfileOptions.SetAccountID(accountID),
```

Comment:
Please add the `includeHistory:true` attribute too. It is setup across the SDKs and we'd like to achieve consistency across SDK example code snippets.